components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultRequestCoordinator.java

```diff
@@ -662,6 +674,9 @@ protected AuthenticationContext initializeFlow(HttpServletRequest request, HttpS
         associateTransientRequestData(request, response, context);
         findPreviousAuthenticatedSession(request, context);
         buildOutboundQueryString(request, context);
+        FrameworkUtils.addAuthenticationRequestToCache(context.getCallerSessionKey(),
```

Comment:
we should be storing the request object to context here. [1]. Can't be able to retrieve it from that without adding a new cache. 


[1] - https://github.com/wso2/carbon-identity-framework/blob/master/components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultRequestCoordinator.java#L508